cmd/keymasterd/app.go

```diff
@@ -1602,6 +1613,8 @@ func main() {
 		serviceMux.HandleFunc(oktaPushStartPath, runtimeState.oktaPushStartHandler)
 		serviceMux.HandleFunc(oktaPollCheckPath, runtimeState.oktaPollCheckHandler)
 	}
+	serviceMux.HandleFunc(bootstrapOtpAuthPath,
```

Comment:
add TODO for only allowing if boostrapOTP is enabled.

Reply:
Done.